tests/testutils/resource_metrics.go

```diff
@@ -43,6 +45,8 @@ const (
 	IntNonmonotonicUnspecifiedSum    MetricType = "IntNonmonotonicUnspecifiedSum"
 )
 
+const buildVersionPlaceholder = "<BUILD VERSION TO BE SET DYNAMICALLY>"
```

Comment:
nit - this is a little verbose imo

```suggestion
const buildVersionPlaceholder = "<FROM_BUILD>"
```